content/community/contributing/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "How to contribute"
+autotoc: false
+---
+<div class='right'><img src="/images/undraw-illustrations/contribute.svg" alt="" width="240" /></div>
+
+If you want to contribute to the Galaxy Project, there are plenty of ways to do it. And here is the good news: __you can contribute with code, but not only with code__!
+
+## Contribute with __code__
+- [__Galaxy Project GitHub Organization__](https://github.com/galaxyproject): repositories of the Galaxy ecosystem, including the [main code of Galaxy](https://github.com/galaxyproject/galaxy).
+
+- __How?__ 
+  - Start by looking at the _CONTRIBUTING.md_ (or _CONTRIBUTING.rst_) file in the repository of your interest.
+  - There are [__Working Groups__](https://galaxyproject.org/community/wg/) focused on particular parts of the ecosystem, you can reach out to them if you want to help.
+- __Where to start?__ Check the issues, pay attention to labels such as _paper-cut_, those are good starting points for new contributors.
+
+## Contribute with __training materials__
+- The [__Galaxy Training Network (GTN)__](https://training.galaxyproject.org/) has an extensive number of [contributors](https://training.galaxyproject.org/training-material/hall-of-fame).
+- __How?__ The GTN has [tutorials on how to contribute](https://training.galaxyproject.org/training-material/topics/contributing/).
+
+## Contribute with __your ideas__
+- The [__Global Galaxy Steering Committee__](https://galaxyproject.org/community/steering/) is a platform for new community initiatives and always eager to listen to yours!
+
+## Contribute by __joining and hosting events__
+- __Engage with the community__. We have [CoFests](https://galaxyproject.org//events/cofests) regularly, the [Galaxy Community Conference (GCC)](https://galaxyproject.org/gcc/) is held every year, and many other events are taking place.
+- Give a workshop, we provide you with the [infrastructure for free](https://galaxyproject.eu/tiaas.html)! Let us know if you do, and we will announce it on the [Events page](https://galaxyproject.org/events/), our newsletter and social media!
```

Comment:
**_Let us know if you do_** > who/where should they reach out to?

Reply:
oh this is a good one, I forgot to link it